.github/workflows/build-and-publish-chart.yml

```diff
@@ -122,3 +120,84 @@ jobs:
           tag_name: ${{ steps.version.outputs.nextVersion }}
           files: cloudzero-agent-${{ env.NEW_VERSION }}.tgz
           make_latest: true
+          target_commitish: ${{ env.COMMIT_HASH }}
```

Comment:
is this a doc attached to each release? I'm not sure I've seen this rendered before on releases

Reply:
🙈 yes! AFAICT. Thus the rollback on using those other bits to get the last and next tag version. I think Kevins action does what we need 🎉